### PR TITLE
cli: fix exception on unestablished connection

### PR DESCRIPTION
**Pull request by lundibundi**
When you made connect but connection was not established yet cli would fail on call and event commands with exception.
Not sure about message in new error so feel free to suggest another one.

### CODE REVIEW

File: lib/cli/command-processor.js [hunk 1]
```diff
@@ -88,12 +88,18 @@ module.exports = class CommandProcessor extends EventEmitter {
   }
 
   call(interfaceName, methodName, args, callback) {
-    if (!this.cli.client) return callback(new Error('Not connected'));
```
Comment (belochub) [inline lib/cli/command-processor.js]: @lundibundi
What do you say about removing redundant indentation by leaving line 91 as it was, and just adding this:
```js
if (!this.cli.connection) return callback(new Error('Connection in progress'));
```

Comment (lundibundi) [inline lib/cli/command-processor.js]: @belochub it doesn't fit, does it?

Comment (belochub) [inline lib/cli/command-processor.js]: @lundibundi, it doesn't fit where?

Comment (lundibundi) [inline lib/cli/command-processor.js]: @belochub done.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/jstp/commit/6cb62a0251f90cb557973f22da1b35ea0c7819f4.

**Pull request closed without merging**